app/models/split_detail.rb

```diff
@@ -3,7 +3,7 @@ class SplitDetail
   include DelegateAttribute
 
   attr_accessor :split
-  delegate_attribute :hypothesis, :assignment_criteria, :description, :owner, :location, :platform, to: :split
+  delegate_attribute :name, :hypothesis, :assignment_criteria, :description, :owner, :location, :platform, to: :split
```

Comment:
should we be able to edit the `name` on this? i don't think we want to make this a `delegate_attribute` but rather just a `delegate`. The difference being that `delegate_attribute` will also delegate the setter allowing changes to be made.